nixos/tests/nginx-cgit.nix

```
@@ -0,0 +1,91 @@
+{
```

That has to be a function. Sorry, that was an oversight on my part.
```suggestion
{ ... }: {
```

---

Once that is fixed and the tests pass I will ping someone to merge.